index.html

```diff
@@ -7,80 +7,86 @@
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <!-- 네비게이션 -->
-    <header class="nav-bar">
-      <a href="'/'" target="_blank">
-        <img  class="logo" src="Property 1=sm.png">
-      </a>
-      <a href="/login" target="_blank">
-        <img class="login-btn" src="btn_small.png">
-      </a>  
-    </header>
-  <!--본문 시작-->
-  <figure>
-    <div class="home-top">
-      <p class="words"> 일상의 모든 물건을<br>거래해 보세요</p>
-      <img src="Img_home_top.png" class="top-image">
-      <a href="/items" target="_blank">
-        <img class="show-more-btn" src="btn_large.png" alt="구경하러 가기">
-      </a>
-    </div>
-  </figure>
-  <figure class="first-p">
-      <img class="hot-item" src="desktop_02.png">
-  </figure>
-  <figure class="second-p">
-      <img class="search-item" src="desktop_03.png">
-  </figure>
-  <figure class="third-p">
-      <img class="resister-item" src="desktop_04.png">
-  </figure>
-<section>
-  <div class="home-bottom">
-    <p class="bottom-word">믿을 수 있는<br> 판다마켓 중고 거래</p>
-    <img src="Img_home_bottom.png" class="bottom-img">
-  </div>  
-</section>
-
-<!--푸터 시작-->
-<footer class="footer-style">
-  <span class="footer-inner">
-    <span class="codeit-2024">@codeit - 2024</span>
-
-    <!--Privacy Policy , FAQ 시작-->
-    <span class="footer-link">
-      <a href="/privacy">
-        <span class="Policy">Privacy Policy</span>
-      </a>
-      <a href="/faq">
-        <span class="FAQ">FAQ</span>
+  <!-- header 시작 -->
+  <header class ="header">
+    <a href= "'/'">
```

Comment:
❗️ 수정요청
오타인 것 같습니다~

```suggestion
    <a href= "/">
```